tools/unix/generate_drules.sh

```diff
@@ -7,6 +7,55 @@ export PYTHONDONTWRITEBYTECODE=1
 OMIM_PATH="${OMIM_PATH:-$(dirname "$0")/../..}"
 DATA_PATH="${DATA_PATH:-$OMIM_PATH/data}"
 
+VALIDATE_ONLY=0
+
+while getopts ":v" option
```

Comment:
Why it can't validate when regenerating styles, and fail if something is wrong, thus forcing the author of changes to fix the issue?

Reply:
This is a good option.
ATM our style-related testing / validation is inconsistent in a sense part of the checks are done in the cpp test suite and another part is in python / kothic.
IMO we need to come up with a more coherent vision first.

Reply:
Sorry, I don't understand, what are the cons of running validation from the script which regenerates drules and fails if something is wrong?

Reply:
If by the "validation" you mean all the styles-related checks we have incl. the ones in the tests suite, then its quite slow (some minutes in my case) and sets a dependency on freshly built test binaries.

So running (and perhaps building) the whole suite for each regeneration is too slow for a styles dev process.

But it'll be great to have as a final step before push. The question is how to ensure each dev runs it in the end while not slowing down each run unnecessarily?

Reply:
Can you please describe in more detail all validation scripts/steps we have now (including this PR), and how long they are running?